htdocs/main.inc.php

```diff
@@ -1719,6 +1719,13 @@ function top_htmlhead($head, $title = '', $disablejs = 0, $disablehead = 0, $arr
 
 		// Mobile appli like icon
 		$manifest = DOL_URL_ROOT.'/theme/'.$conf->theme.'/manifest.json.php';
+		$parameters = array('manifest'=>$manifest);
+		$resHook = $hookmanager->executeHooks('setManifest', $parameters); // Note that $action and $object may have been modified by some hooks
```

Comment:
For new introduced hook we must now start the method name with hook... Can you rename into
hookSetManifest